lock/gremlin_consul_comb_lock.go

```diff
@@ -65,7 +65,7 @@ func (lock ConsulCombinationLock) Unlock() error {
 	if err != nil {
 		errResponse = fmt.Sprintf("%s. Local unlock error: %v", errResponse, err)
 	}
-	if err != nil {
+	if errResponse != "" {
```

Comment:
soo you're not returning the err if ConsulLock.Unlock fails ... because you still want to attempt the LocalLock.Unlock?

Reply:
Yeah, because I don't want to return an error on Consul lock fails, have the request return and then not have the local lock unlocked. It could cause a mutex we can't unlock.